literals/rand.go

```diff
@@ -0,0 +1,34 @@
+package literals
+
+import (
+	"crypto/rand"
+	"fmt"
+	mathrand "math/rand"
+	"os"
+	"strings"
+)
+
+var envGarbleSeed = os.Getenv("GARBLE_SEED")
+
+// If math/rand.Seed() is not called, the generator behaves as if seeded by rand.Seed(1),
+// so the generator is deterministic.
+
+// genRandBytes return a random []byte with the length of size.
+func genRandBytes(size int) []byte {
+	buffer := make([]byte, size)
+
+	if strings.HasPrefix(envGarbleSeed, "random;") {
+		_, err := rand.Read(buffer)
+		if err != nil {
+			panic(fmt.Sprintf("couldn't generate random key:  %v", err))
+		}
+	} else {
+		mathrand.Read(buffer) // error is always nil so save to ignore
```

Comment:
s/save/safe/

Reply:
also, if it is safe to ignore, I would still panic, just for the sake of being conservative